spyder/plugins/completion/languageserver/client.py

```diff
@@ -71,7 +71,7 @@ class LSPClient(QObject, LSPMethodProviderMixIn):
 
     #: Signal to warn the user when either the transport layer or the
     #  server are down
-    sig_lsp_down = Signal(str)
+    sig_down = Signal(str)
```

Comment:
Hmm this is a weird name, what do you want to convey @ccordoba12 ?

Reply:
`sig_stopped` 
`sig_errored`

Reply:
Yeah, I'm not happy with it either. It's used to inform the LSP plugin that either the transport layer or the server are down. Could you come up with a better name for it?

Reply:
What about `sig_is_down`? Although `sig_errored` is good for me too.

Reply:
Another option could be `sig_error_occurred` (as the Qt signal).

Reply:
I decided to go for `sig_went_down`.

Reply:
hehe ok, that one works!